plugwise/helper.py

```diff
@@ -123,6 +123,14 @@ async def _request(
                 resp = await self._websession.get(
                     url, headers=use_headers, auth=self._auth
                 )
+            if method == "post":
```

Comment:
Maybe we should consider (not for this PR!) a rewrite to match/case for this construct

Reply:
Sure that's a good idea :)